ui/src/components/hintDataSelect.tsx

```diff
@@ -11,7 +11,7 @@ import React from "react";
 import { isValid } from "util/valid";
 
 export type Selection = {
-  value: DataValue;
+  value: DataValue | string;
```

Comment:
This shouldn't be needed as `DataValue` already includes type `string` ([types.ts](https://github.com/DataBiosphere/tanagra/blob/main/ui/src/data/types.ts#L5C1-L5C74)):
```
export type DataValue = null | string | number | bigint | boolean | Date;
```